java-sdk/example/data/test_cb_mod.json

```diff
@@ -0,0 +1,19 @@
+{
+  "engineId": "test_cb",
+  "engineFactory": "com.actionml.templates.cb.CBEngine",
+  "comment": "add mirrorContainer to point to a location on your system and enable mirroring to that location",
+  "mirrorType": "localfs",
+  "mirrorLocation": "/Users/pat/big-data/harness/mirrors/cb_test",
+  "comment": "add sharedDBName for sharing a DBs for Users, should be named with lowercase and underscore separator, aka snake case",
+  "sharedDBName": "harness_test_shared_users",
+  "comment": "modelContainer is output an should be a full directory path on your local fs",
+  "modelContainer": "/Users/pat/big-data/harness/models/",
+  "algorithm":{
+    "maxIter": 100,
+    "regParam": 0.0,
+    "stepSize": 0.1,
+    "bitPrecision": 24,
+    "comment": "modelContainer is output an should be a full directory path on your local fs",
+    "maxClasses": 3
+  }
+}
```

Comment:
This is a test engine config that can be used with `harness update test_cb_mod.json` but requires there already be an engine of the same id: `test_cb`

Reply:
not important, you can ignore